tests/test/storage/test_file_descriptor.cpp

```diff
@@ -10,18 +11,12 @@
 using namespace storage;
 
 namespace tests {
-FileDescriptor& getRootFd()
+
+TEST_CASE("Test stat and mkdir", "[storage]")
```

Comment:
All of the tests in this file were called `"Check..."` but `"Test..."` is more common in the codebase.